frontend/src/app.tsx

```diff
@@ -12,13 +13,40 @@ import { GlobalProvider } from './contexts/global/global-provider'
 import { createLogger, initLogger } from './lib/logging'
 import { AppRouter } from './routes'
 import { Service } from './service'
+import { BrowserOpenURL } from './wailsjs/runtime'
 
 const logger = createLogger('GlobalActions')
 
+const getAnchor = (element: HTMLElement | null): HTMLAnchorElement | null => {
+  if (element?.nodeName.toLocaleLowerCase() === 'a') {
+    return element as HTMLAnchorElement
+  }
+
+  if (element?.parentNode?.nodeName.toLowerCase() === 'a') {
+    return element.parentNode as HTMLAnchorElement
+  }
+
+  return null
+}
+
 /**
  * Renders all the providers
  */
 function App() {
+  useEffect(() => {
```

Comment:
question (non blocking): Why use this over just using the ExternalLink component?

Reply:
my idea was that this will be most likely needed only inside an electron environment. once i get to move the wallet ui into a serparate lib, i'd like the apps to manage these platform specific things themselves, which would probably happen somewhere here. initially based on some wails thread here:  https://github.com/wailsapp/wails/discussions/1898